### ISSUE TITLE
Add kebab-case for file names

### ISSUE THREAD

**Issue by aschmidt93** [odata2ts/odata2ts#286]
Hi there,

I'd like to suggest adding another option to `NamingConventions` for file names: `kebab-case` as this is a popular casing for TS projects.

**Comment by aschmidt93** [comment]
You're right, the file names use the names of the .xml input files.

**Comment by texttechne** [comment]
Actually, I always wanted to do that for file names. However, I need to look into it. Maybe it won't be configurable... but will be coming for file names only.

**Comment by texttechne** [comment]
Little correction: It was only the folder names which used `snake-case` when `bundledFileGeneration: false`.
So I've switched there to `kebab-case` for folder names only.

Released with [@odata2ts/odata2ts@0.37.0](https://github.com/odata2ts/odata2ts/releases/tag/%40odata2ts%2Fodata2ts%400.37.0).

Sorry, that was what I had in mind. But needed to switch to ESM first to use `change-case`. 

Does this solve your issue? Did you have anything different in mind?
